src/main/java/org/folio/des/service/JobExecutionService.java

```diff
@@ -70,8 +73,13 @@ public JobCommand prepareResendJobCommand(Job job) {
     ExportConfig config = defaultModelConfigToExportConfigConverter.convert(modelConfiguration);
     Optional.ofNullable(config.getExportTypeSpecificParameters()).
       map(ExportTypeSpecificParameters::getVendorEdiOrdersExportConfig)
-        .ifPresent(vendorEdiOrdersExportConfig ->
-          params.put(EDIFACT_ORDERS_EXPORT_KEY, new JobParameter(String.valueOf(vendorEdiOrdersExportConfig))));
+        .ifPresent(vendorEdiOrdersExportConfig -> {
+          try {
+            params.put(EDIFACT_ORDERS_EXPORT_KEY, new JobParameter(objectMapper.writeValueAsString(vendorEdiOrdersExportConfig)));
+          } catch (JsonProcessingException e) {
```

Comment:
Lets use SneakyThrows annotation as in other places in this module. This SneakyThrows runtime exception will be handled in ControllerAdvice